DynamicAnalyzer/views/android/environment.py

```diff
@@ -115,7 +142,7 @@ def set_global_proxy(self, version):
         if version < 5:
             proxy_ip = get_proxy_ip(self.identifier)
         else:
-            proxy_ip = '127.0.0.1'
+            proxy_ip = settings.PROXY_IP
```

Comment:
The Proxy IP is doesn't have to be dynamic for android version >=5.
We make use of adb reverse tcp (https://blog.grio.com/2015/07/android-tip-adb-reverse.html) to make proxy available to device via localhost. So it is static.

Reply:
You are right with your setup, but as I am using docker, the container is attached to a dedicated docker network, not having mobfs and the android emulator on the same host. So making it default to localhost in the config would be a default and changeable to other setups.

Reply:
So currently this setup won't work with non x86 VMs, device or emulators. This require changes to the frida binary.